addons/pos_restaurant/static/src/overrides/components/product_screen/actionpad_widget/actionpad_widget.xml

```diff
@@ -9,14 +9,25 @@
         <xpath expr="//button[hasclass('validation')]" position="after">
             <button
                 t-if="this.swapButton"
-                t-attf-class="submit-order w-100 h-100 {{getMainButtonClasses()}}"
+                t-attf-class="submit-order h-100 {{getMainButtonClasses()}}"
                 t-att-class="swapButtonClasses"
-                t-on-click="submitOrder">
+                t-on-click="submitOrder"
+                style="width: 170px;">
                 <i class="fa fa-cutlery"></i>
                 Order
-                <div class="break-line">
-                    <t t-esc="categoryCount" />
+                <div t-if="displayCategoryCount.length" class="break-line fw-normal fs-6 w-100  border-top mt-1 pt-1">
+                    <t t-foreach="displayCategoryCount" t-as="categoryCountLine"  t-key="categoryCountLine_index">
+                        <t t-if="!categoryCountLine.more">
```

Comment:
Is this condition really needed? What is this `more` props?

Reply:
No, it was from a previous solution, too much changes I forgot to remove it